ReplaceServiceBusConnectionString.ps1

```diff
@@ -0,0 +1,62 @@
+<#
+
+.SYNOPSIS
+
+Allows you to update service bus connection strings in the directory (and sub-directories) supplied.
+
+.DESCRIPTION
+
+Updates the service connection string in .config and .json files 
```

Comment:
Can we add the Syntax and an Example?

Reply:
If you run it, there's default syntax which I believe is enough of an example. But if you want more of an example I could do that, let me know